README.md

```diff
@@ -90,7 +90,7 @@ Platforms](https://github.com/HerculesWS/Hercules/wiki/Supported-Platforms) wiki
     - [TortoiseGit](https://tortoisegit.org/) (Windows)
 
 
-Installation
+<a name="Installation">Installation</a>
```

Comment:
Manually adding anchors isn't necessary: GitHub already adds anchors in the form of `user-content-{name}` so here there's already a `user-content-installation` anchor